HTTP Server.core/Request.cs

```diff
@@ -0,0 +1,40 @@
+﻿ namespace HTTPServer.core
+{
+    public class Request
+    {
+        public string EntireMessage { get; }
+        public string HttpVersion { get; }
+        public  string Uri { get; }
+        public  string Method { get; }
+        public string Body { get; }
+
+        public Request(string requestMessage)
```

Comment:
I'll answer all the string questions here.  It is fine for HttpVersion, Uri, and Method to be strings, because it makes them easier to parse and check for the response.  The rest should be byte[].  I had not considered images for some reason.